_posts/yyyy-mm-dd-ansible-czy-da-sie-uporzadkowac-chaos.md

```diff
@@ -0,0 +1,301 @@
+---
+layout:    post
+title:     "Ansible - czy da się uporządkować chaos"
+date:      YYYY-MM-DD HH:mm:ss +0100
+published: false
+author:    rmastalerek
+image:     /assets/img/posts/yyyy-dd-mm-ansible-czy-da-sie-uporzadkowac-chaos/ansible.png
+tags:
+    - ansible
+    - automation
+    - devops
+---
+
+# tl;dr
```

Comment:
Tutaj czysto techniczna sprawa - zrezygnowaliśmy z koncepcji zaczynania wpisu od nagłówka, bo jego treść pojawia się wtedy w "skrócie" wpisu na głównej stronie. To nie wygląda za fajnie